books/statistics/bake

```diff
@@ -0,0 +1,182 @@
+
+gemfile do
+  gem 'openstax_kitchen', ENV['USE_LOCAL_KITCHEN'] ? { path: '/code/kitchen' } : '12.0.0'
+  gem 'slop', '4.8.2'
+  gem 'byebug'
+end
+
+recipe = Kitchen::BookRecipe.new(book_short_name: :statistics) do |doc|
+  include Kitchen::Directions
+
+  # Set overrides
+  doc.selectors.override(
+    reference: '.references'
+  )
+
+  book = doc.book
+  metadata = book.metadata
+
+  # Some stuff just goes away
+  book.search('cnx-pi').trash
+  book.body.search('div[data-type="metadata"] > div[data-type="description"]').trash
+
+  BakePreface.v1(book: book, title_element: 'h1')
+  BakeInlineLists.v1(book: book)
+  BakeUnclassifiedNotes.v1(book: book)
+  BakeChapterTitle.v1(book: book)
+  BakeChapterIntroductions.v2(
+    book: book,
+    strategy_options: {
+      strategy: :default,
+      bake_chapter_outline: false,
+      introduction_order: :v1
+    }
+  )
+
+  book.chapters.each do |chapter|
+    BakeLearningObjectives.v1(chapter: chapter)
+
+    # EOC sections
+    BakeChapterGlossary.v1(chapter: chapter, metadata_source: metadata)
+    BakeChapterSummary.v1(chapter: chapter, metadata_source: metadata)
+
+    MoveCustomSectionToEocContainer.v1(
+      chapter: chapter,
+      metadata_source: metadata,
+      container_key: 'formula-review',
+      uuid_key: '.formula-review',
+      section_selector: 'section.formula-review',
+      append_to: chapter
+    ) do |section|
+      RemoveSectionTitle.v1(section: section)
+      section.prepend(child: EocSectionTitleLinkSnippet.v1(
+        page: section.ancestor(:page)
+      ))
+    end
+
+    MoveCustomSectionToEocContainer.v1(
+      chapter: chapter,
+      metadata_source: metadata,
+      container_key: 'practice',
+      uuid_key: '.practice',
+      section_selector: 'section.practice',
+      append_to: chapter
+    ) do |section|
+      RemoveSectionTitle.v1(section: section, selector: 'h3')
+      section.prepend(child: EocSectionTitleLinkSnippet.v1(
+        page: section.ancestor(:page)
+      ))
+    end
+
+    MoveCustomSectionToEocContainer.v1(
+      chapter: chapter,
+      metadata_source: metadata,
+      container_key: 'bring-together-exercises',
+      uuid_key: '.bring-together-exercises',
+      section_selector: 'section.bring-together-exercises',
+      append_to: chapter
+    ) do |section|
+      RemoveSectionTitle.v1(section: section)
+    end
+
+    BakeFreeResponse.v1(chapter: chapter, metadata_source: metadata)
+
+    MoveCustomSectionToEocContainer.v1(
+      chapter: chapter,
+      metadata_source: metadata,
+      container_key: 'bring-together-homework',
+      uuid_key: '.bring-together-homework',
+      section_selector: 'section.bring-together-homework',
+      append_to: chapter
+    ) do |section|
+      RemoveSectionTitle.v1(section: section)
+    end
+
+    BakeChapterReferences.v1(chapter: chapter, metadata_source: book.metadata)
+
+    exercise_selectors = '.practice, .bring-together-exercises, .free-response, .bring-together-homework'
```

Comment:
didn't see these yesterday -- could you add tag names (I assume these are `section`s) to speed up the selectors? :)

Reply:
Done!